diff --git a/examples/jsm/controls/OrbitControls.js b/examples/jsm/controls/OrbitControls.js
index dc3b42b56a..518fc543e1 100644
--- a/examples/jsm/controls/OrbitControls.js
+++ b/examples/jsm/controls/OrbitControls.js
@@ -161,6 +161,12 @@ class OrbitControls extends EventDispatcher {
 
 		};
 
+		this.resetOrthoPanZoom = function () {
+         panOffset.set(0,0,0);
+         scope.object.zoom = 1;
+         scope.object.updateProjectionMatrix();
+      };
+
 		// this method is exposed, but perhaps it would be better if we can make it private...
 		this.update = function () {
 
