diff --git a/examples/jsm/controls/OrbitControls.js b/examples/jsm/controls/OrbitControls.js
index 7360aab9b3..8862897d32 100644
--- a/examples/jsm/controls/OrbitControls.js
+++ b/examples/jsm/controls/OrbitControls.js
@@ -289,6 +289,14 @@ class OrbitControls extends Controls {
 
 	}
 
+	resetOrthoPanZoom() {
+
+		this._panOffset.set(0, 0, 0);
+		this.object.zoom = 1;
+		this.object.updateProjectionMatrix();
+
+	}
+
 	update( deltaTime = null ) {
 
 		const position = this.object.position;
